feature-examples/python/api_key/list_api_key.py

```diff
@@ -0,0 +1,91 @@
+"""Example of using the Vectara REST API to list the API Keys."""
+
+from dataclasses import dataclass
+import logging
+from typing import Union
+
+import requests
+
+
+@dataclass
+class CorpusData:
+    """CorpusData class."""
+
```

Comment:
The comment does not add anything beyond what the code shows, so could be removed or else embellished with information about what the class represents ("Corpus id and name.", for example). If keeping, it could document the class attributes.
https://google.github.io/styleguide/pyguide.html#384-classes